### PR TITLE
Fix markdown linter errors

### PR DESCRIPTION
**Pull request by anniepamela2020**
@bradbeck @trmiller @mlieberman85 

Please review this PR

### CODE REVIEW

**Review by trmiller — CHANGES_REQUESTED** [review]
Few things here

- Commit message needs to be much more helpful.  See [contributing doc](https://thesecuresoftwarefactory.github.io/ssf/docs/contributing/how-to-contribute/) for some guidance
-  Don't add the .DS_Store - that is a mac system file.  Please update the .gitignore as well.
-  There are a few changes in here reverting previous PR's.  In particular on 
    - kyverno
    - jq typo fix

File: platform/vendor/vendor_kyverno.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# KYVERNO_LATEST_STATUS=$?
+# KYVERNOPRE_LATEST=`COSIGN_REPOSITORY=$COSIGN_REPOSITORY cosign verify --key $PUBKEY ghcr.io/kyverno/kyvernopre:$KYVERNO_TAG_RELEASE`
+# KYVERNOPRE_LATEST_STATUS=$?
+# KYVERNOCLI_LATEST=`COSIGN_REPOSITORY=$COSIGN_REPOSITORY cosign verify --key $PUBKEY ghcr.io/kyverno/kyverno-cli:$KYVERNO_TAG_RELEASE`
+# KYVERNOCLI_LATEST_STATUS=$?
+
+# if [ $KYVERNO_LATEST_STATUS -eq 0 ] && [ $KYVERNOPRE_LATEST_STATUS -eq 0 ] && [ $KYVERNOCLI_LATEST_STATUS -eq 0 ] ; then
+#   # Download the release file
+  curl "$KYVERNO_URL" > $KYVERNO_INSTALL_DIR/release.yaml
+#fi
```
Comment (bradbeck) [inline platform/vendor/vendor_kyverno.sh:28]: This file was removed as part of #95. Was this intentionally re-added?

File: platform/vendor/vendor.yaml [hunk 2]
```diff
@@ -3,13 +3,7 @@ files:
...
-    validation_type: "rekor"
   - release_file: "https://storage.googleapis.com/tekton-releases/chains/previous/v0.5.0/release.yaml"
     rekor_uuid: "3a62d47dcbe0727513ac2e2dcc3a41bfd413ebc128bb661ed4f115d4db83200f"
     destination_dir: "tekton/chains"
     version: "v0.5.0"
-    validation_type: "rekor"
-  - release_file: "https://raw.githubusercontent.com/kyverno/kyverno/main/config/install.yaml"
-    destination_dir: "kyverno/release"
-    sha256: "f20dbfd87a91935f8b69b9a9802edbc486a1c4a09861b49cc12c79b357202ecc"
-    validation_type: "sha256"
```
Comment (bradbeck) [inline platform/vendor/vendor.yaml:15]: This appears to be reverting changes from #95.

File: platform/vendor/vendor.sh [hunk 3]
```diff
@@ -1,10 +1,6 @@
...
 set -euo pipefail
 
-GIT_ROOT=$(git rev-parse --show-toplevel)
+go install github.com/trmiller/vendorme@91071b4ac30f03e42774f72ebf3fd80f57a710ea
 
-go install github.com/trmiller/vendorme@0b7091a0736be19ca2fd0a3245d997356822ba07
-
-pushd "$GIT_ROOT/platform/vendor"
 vendorme pull
-popd
```
Comment (bradbeck) [inline platform/vendor/vendor.sh:10]: This appears to be reverting changes from #95.

File: platform/30-kyverno-setup.sh [hunk 4]
```diff
@@ -34,7 +34,7 @@ DOCKER_CONFIG_JSON=$HOME/.docker/config.json
 #   Installation: https://kyverno.io/docs/installation/
 
 echo -e "${C_GREEN}Installing Kyverno...${C_RESET_ALL}"
-kubectl apply -f "$KYVERNO_INSTALL_DIR"/install.yaml
+kubectl apply -f "$KYVERNO_INSTALL_DIR"/release.yaml
```
Comment (bradbeck) [inline platform/30-kyverno-setup.sh:37]: This appears to be reverting changes from #95.

File: platform/vendor/kyverno/release/release.yaml [hunk 5]
```diff
@@ -7360,6 +7360,7 @@ metadata:
     app.kubernetes.io/name: kyverno
     app.kubernetes.io/part-of: kyverno
     app.kubernetes.io/version: latest
+    app.kubernetes.io/ownerreference: "true"
```
Comment (bradbeck) [inline platform/vendor/kyverno/release/release.yaml:7363]: This appears to be reverting changes from #95.

File: Makefile [hunk 6]
```diff
@@ -72,3 +72,7 @@ docs-serve: ## Serve the site locally with hot-reloading
 .PHONY: docs-build
 docs-build: ## Build the documentation site
 	cd docs && zola build
+
+.PHONY: linter-markdown
+linter-markdown: ## Lint markdown files
+	npx markdownlint-cli2  "**/*.md" "#docs"
```
Comment (bradbeck) [inline Makefile:78]: Do we need to add `npm` & `npx` as prerequisites somewhere?

**Review by bradbeck — APPROVED** [review]

File: Makefile (hunk 6 above)
Comment (pxp928) [reply Makefile:78]: I am thinking this should be a github action that happens automatically. I dont think we want to install more tools. What do others think?

```
jobs:
  lint-changelog:
    name: Lint changelog file
    runs-on: ubuntu-latest
    steps:
    - name: Check out code
      uses: actions/checkout@v2

    - name: Lint changelog file
      uses: docker://avtodev/markdown-lint:v1 # fastest way
      with:
        rules: '/lint/rules/changelog.js'
        config: '/lint/config/changelog.yml'
        args: './CHANGELOG.md'
        ignore: './one_file.md ./another_file.md' # multiple files must be separated with single space

    # Or using current repository as action:

    - name: Lint changelog file
      uses: avtodev/markdown-lint@v1
      with:
        rules: '/lint/rules/changelog.js'
        config: '/lint/config/changelog.yml'
        args: './CHANGELOG.md'
```

Comment (rgreinho) [reply Makefile:78]: It will be a GitHub action, but we must be able to run it locally as well.

Comment (pxp928) [reply Makefile:78]: Ah ok cool.

File: tools/install-ci.sh [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+# Detect the platform.
+PLATFORM=$(uname)
+
+# Install packages if needed.
+echo -e "${C_GREEN}Installing packages if needed...${C_RESET_ALL}"
+case "${PLATFORM}" in
+  Darwin)
+  node --version || brew install node
+  ;;
+esac
```
Comment (bradbeck) [inline tools/install-ci.sh:17]: A newline at the end here would be good.

**Comment by bradbeck** [comment]
I kind of feel like the addition of the YAML linting goes beyond the original intent of this PR. May the YAML changes would be better in their own PR. It also helps to keeps the PRs a bit smaller, which helps when trying to review.

**Review by trmiller — APPROVED** [review]

**Review by bradbeck — APPROVED** [review]

**Pull request merged**